db/database.rb

```diff
@@ -7,32 +7,33 @@ def initialize
     @users = @db[:users]
   end
 
-  def increase_karma(user_name)
-    user = users.where(user_name: user_name)
+  def increase_karma(user_name, chat_id)
+    user = users.where(user_name: user_name, chat_id: chat_id)
```

Comment:
Does it work with users, which already exist and do not have `chat_id`?

Reply:
Now, but for now it used only in one chat, so i planned to just make a sql query which will fix it

Reply:
But i can create a data migration for it or rake task
What do you think, @yashka713 ?

Reply:
I work on it.
The best way is creating `command` for admin users.
By this command, the admin will update user fields.
For example, by replaying for the message with a command.

<img width="213" alt="Screen Shot 2019-07-17 at 21 44 10" src="https://user-images.githubusercontent.com/15985355/61402545-0adc6c00-a8dc-11e9-8266-869e7befd03a.png">

This allows you to get information about the user, check that command started by admin, and update user fields.
If you find a better way write it [here](https://github.com/TheBlackArroVV/super_karma_bot/issues/5)